README.rst

```diff
@@ -160,4 +165,52 @@ contributing
 
 On github create pull requests using a named feature branch.
 
+.. _backers
+
+backers
+-------
+
+
+[Become a backer]((https://opencollective.com/doit#backer)) and show your support to our open source project.
+
+.. image:: https://opencollective.com/webhook/backers.svg?width=890
```

Comment:
My project is NOT `webhook`.

Reply:
I can fix this myself. not a big deal. I just realized this might have been manually added.